istioctl/pkg/validate/validate.go

```diff
@@ -288,6 +288,7 @@ func validateFiles(istioNamespace *string, defaultNamespace string, filenames []
 		if err != nil {
 			errs = multierror.Append(errs, err)
 		}
+		reader.Close()
```

Comment:
what does close of stdin do?

Reply:
Updated. I think it doesn’t matter if Stdin is closed after reading.